### PR TITLE
Set Up Hello Microverse Project

### PR DESCRIPTION
**Pull request by amel129**
Create a .gitignore file.
Set up a linter for HTML and CSS 
Add the README File 
Add 3 files HTML with the head "hello Microverse", CSS, and test.md

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (osamaashraf6) [inline .gitignore]: - [ ] While you make a great effort I see that your ```test.md``` file is existing in GitHub and this is not expected, so kindly we expect that you put the ```test.md``` file in the gitignore, so when you push the changes to the GitHub the ```test.md``` file will not be in the GitHub as you put it in the gitignore this is like you did with node_modules folder, so : 

- Please try to put the ```test.md``` file in the gitignore to not pushing it in the GitHub, so with that you will stick to the requirements 😊💯
- Also please consider renaming the file from the ```text.md``` to be ```test.md``` as the requirements told you to create a new file called ```test.md``` with that you will stick to the requirements 🚀💯

image

File: README.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
```
Comment (osamaashraf6) [inline README.md]: - [ ] You did a great effort 😃 at using the Readme template from microverse, but kindly make sure placeholders are replaced with your project and personal data as you see here you didn't update the data of the Readme with your information, also please remember. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your Readme | Example professional Readme| 
| --- | --- |
|  image   |    image  |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @amel129 ☺️, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 


you-are-near

## Highlights ✔️

- [x] Descriptive pull request title 
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️

- [x]  You did a great effort at writing the commits, but I see that there is one commit not professional, so Please 😊 consider using best practices when you write commit messages in future commits as you can not now update the old commits just in the future commit try to make it professional, they should be capitalized sentences in the imperative form, for example: "Fix bug at CSS file" instead of "Bug fixed in CSS file or Fixed Bug" ... etc 👍

 
image



- Check the comments under the review._

## Optional suggestions
- N/A

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @amel129 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great readme file

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**